### PR TITLE
Extension support

### PR DESCRIPTION
**Pull request by kivikakk**
Add support for extensions in `commonmark/extensions` options key. These are already available as of `commonmarker==0.14.0` so no changes to gemspec needed.

### CODE REVIEW

File: lib/jekyll-commonmark.rb [hunk 1]
```diff
@@ -12,17 +12,30 @@ def initialize(config)
           @options = [:DEFAULT]
         else
           @options.reject! do |e|
-            unless CommonMarker::Config::Parse.keys.include? e.to_sym
+            unless CommonMarker::Config::Parse.keys.include? e
```
Comment (benbalter) [inline lib/jekyll-commonmark.rb]: :+1: On removing the `to_sym` call. This could be used as a DoS vector.

Comment (kivikakk) [inline lib/jekyll-commonmark.rb]: We already call `to_sym` on the entirety of what becomes `@options` above, so technically there's still a DoS vector. (This is replicated below.)

**Comment by pathawks** [comment]
Thanks @kivikakk :beers:

I would be happy to transfer this repo to the @jekyll organization if there is interest :+1:

**Comment by parkr** [comment]
@pathawks maybe for jekyll v3.6, we can ship support for commonmark in core if our tests look good.

**Comment by pathawks** [comment]
> I would be happy to transfer this repo to the @jekyll organization if there is interest

Just an update for those watching this PR: I have transferred this repo to the @jekyll organization. :+1:

**Pull request merged**